docs/proposals/accepted/observatorium.md

```diff
@@ -0,0 +1,222 @@
+
+
+
+2. Focus on a single Observability signal instead of many e.g rely on traces only.
+
+There is a huge debate ([example](https://twitter.com/el_bhs/status/1349406398388400128)) in open source what Observability signal is the most important and which one we can drop to reduce the complexity and cost of achieving observability goals. The popular statement is to trace everything and deduce metrics or even logs from traces. Or log only and deduce metrics and traces from log lines. All of those are novel proposals, but we believe in **no fit-all solution**. From our experience, healthy infrastructure should have, first of all, a metric collection pipeline for real-time debugging and reliable alerting. Only if metrics are achievable, which solves ~90% of your monitoring and observability use cases, one can focus on installing logging infrastructure, which solves further ~90% of the remaining 10% of use cases. Only if that is met, tracing and profiling might help for more advanced use cases and might be required for the remaining ~1% of Observability cases. So does it make sense to focus on tracing only? Might make sense for some, might be over-engineering and over-expensive for others.
+
+So in our opinion, the biggest value is in a system that gathers all metrics, logging, tracing, and profiles (if you choose to) and allows for a correlation between those.
+
+
+
+3. Don’t do it. Paid, close source solutions exist.
+
+There are many players aiming to solve similar goals, but in the close source: Tanzai, Splunk, DataDog, ElasticCloud, StackDriver (GCP), CloudWatch (AWS), AWS Prometheus Managed, Grafana Enterprise, Logz.io, Honeycomb.io (there are more, those are just examples). Why investing time in contributing, developing, and running Observatorium if there are good paid services?
+
+There are a few reasons:
+
+
+
+*   **Cost: **
+
+All of the above are paid. Of course, running Observatorium is far from free. However, it allows you to make the decision if you want to run yourself or allow others to manage Observatorium as SaaS for you. (While currently, no SaaS option emerged, because we announced the project now, we expect many will emerge). Additionally, one of the Observatorium’s main goals is to limit operational (maintenance) and running costs to a minimum.
+
+
+
+*   **Missing features or relying on one signal: **
+
+Most of the mentioned solutions focus only on part of the observability story. This is great, however, it might mean inconsistency and further complexity if you have to pay for logging to service X, use metrics with project Y, and pay to service Z to have traced. See
```

Comment:
```suggestion
Most of the mentioned solutions focus only on part of the observability story. This is great, however, it might mean inconsistency and further complexity if you have to pay for logging to service X, use metrics with project Y, and pay to service Z to have traces. See
```